scripts/exregional_run_analysis.sh

```diff
@@ -179,7 +187,7 @@ stampcycle=$(date -d "${START_DATE}" +%s)
 minHourDiff=100
 loops="009"    # or 009s for GFSv15
 ens_type="nc"  # or nemsio for GFSv15
-foundens=false
+foundens=".false."
```

Comment:
Bash does not evaluate strings to boolean values. Please use an easier value here. I suggest staying with `true` and `false`.

Reply:
Made the change.

Reply:
It does not seem to have been made.

Reply:
Sorry , forgot to push the change back.
Please check again.